src/main/java/core/basesyntax/Ball.java

```diff
@@ -0,0 +1,16 @@
+package core.basesyntax;
+
+public class Ball {
+    private final String color;
```

Comment:
I would say, making color variable of type `Color` would be beneficial from readability perspective, but that is minor.